electrode_models/conversion_electrode.py

```diff
@@ -1,114 +1,188 @@
 """
-    conversion_electrode.py
+    sulfur_electrode.py
```

Comment:
Can we keep this as `conversion_electrode`?  I like having it be more generally applicable.